pages/dug_10/x-dasharo-dbx-uc-ci.md

```diff
@@ -0,0 +1,196 @@
+---
+theme: ../../slidev-template/theme
+layout: cover
+background: /intro.png
+class: text-center
+
+---
+
+## Enhancements in Dasharo CI: Automatic DBX and microcode refresh
+
+---
+
+## Agenda
+
+- Introduction
+- DBX and microcode
+- Automations
+- Closing thoughts
+
+---
+
+## Introduction
+
+- Michał Kopeć
+- Firmware Engineer working primarily with coreboot and EDK2 but also Heads and
+  Linux
+- Have been at 3mdeb for 4 years now
+- Free and open source software enthusiast
+- ThinkPad collector
```

Comment:
This slide asks for some polishing. Please check how my slides look like from mastering uefi secure boot training. Benefit of such slide is that you make it one in some time and reuse, so those slides are high value.

![Screenshot 2025-06-10 at 09-27-24 Mastering UEFI Secure Boot and Intel Root of Trust Technologies - Slidev](https://github.com/user-attachments/assets/3df13faf-3874-4c2a-a315-258aed299ec8)